frontend/taipy-gui/src/components/Taipy/Chart.tsx

```diff
@@ -488,12 +550,52 @@ const Chart = (props: ChartProp) => {
         props.figure,
     ]);
 
+    const updateDateVarsName = useMemo(() => {
```

Comment:
There's a function for this ...
 we only need the backend variable name for animationData => use `getUpdateVar(updateVars, "animationData")` from utils.ts.